doc/content.tex

```diff
@@ -1,677 +0,0 @@
-% Author: Pavol Loffay
-% Project: Master thesis - Hawkular alert prediction
-% Date 20.2.2015
-
-% Cross-Referencing Conventions:
-% chap: chapter
-% sec: section
-%
-% appen: everything from appendix, e.g. appen:img:foo
-%
-% eq: equation
-% img: image
-% tab: table
-% alg: algorithms
-% item: item in itemize
-%
-% example \label{img:foo-foo}
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Introduction} \label{chap:introduction}
-When driving successful business on the internet, it is important to assure application`s health and reliability. One
-can achieve that by monitoring subjected resources and setting up clever alerting system.
-
-These features are offered by many monitoring systems, however being predictive in this area can even prevent
-undesirable states and most importantly gives administrators more time to react to such events. For instance it
-can decrease downtime of an application or ability to load balance workload in advance by horizontal scaling of
-targeted services.
-
-Alerting systems are sophisticated and can be composed by many conditions. This work focuses on predicting future
-metric values which are then sent as input for evaluation to alerting system.
-
-The work starts with time series theory by describing various approaches for time series modelling and forecasting.
-The chapter describes models which are used within the work, could be used or explains why the decision was made to
-use other models.
-
-Following chapter demonstrates models from previous chapter on real time series. It visually shows predictive
-capabilities of targeted models and compares produced statistical quantities.
-
-The third chapter briefly describes existing monitoring and management solutions which are to some extend
-Hawkular competitors. Java libraries for time series forecasting are also mentioned there.
-
-The fourth chapter describes implementation part of the thesis. It starts with core artifacts for time series
-forecasting and continues to building web application and integration into Hawkular.
-
-The last chapter is dedicated to the evaluation of the forecasting accuracy of the implemented solution. As a default
-implementation were chosen models from \texttt{forecast} package from language R.
-
-    %%%%%%%%
-    \section{Hawkular} \label{sec:hawkular}
-    The implementation part of the master's thesis is developed as a part of an open source project
-    Hawkular\footnote{Available at \url{http://www.hawkular.org}}. Therefore the application architecture and used
-    technologies had to fit into the overall project design.
```

Comment:
..., therefore, ... / ciarka a spojit vety  (hmm, to je divne, aj pri tomto komente mam pocit, ze uz som ho pisala. ale moznoze to bol len nejaky jemu podobny).